doc/Alerting/Templates.md

```diff
@@ -304,14 +304,28 @@ Rule: @if ($alert->name) {{ $alert->name }} @else {{ $alert->rule }} @endif <br>
 {{ $key }}: {{ $value['string'] }}<br>
 @endforeach 
 @if ($alert->faults) <b>Faults:</b><br>
-@foreach ($alert->faults as $key => $value)<img src="https://server/graph.php?device={{ $value['device_id'] }}&type=device_processor&width=459&height=213&lazy_w=552&from=end-72h><br>
+@foreach ($alert->faults as $key => $value)<img src="https://server/graph.php?device={{ $value['device_id'] }}&type=device_processor&width=459&height=213&lazy_w=552&from=end-72h"><br>
 https://server/graphs/id={{ $value['device_id'] }}/type=device_processor/<br>
 @endforeach 
 Template: CPU alert <br>
 @endif
 @endif
 ```
 
+#### MS Teams formatted default template:
```

Comment:
Why is this ms teams specific?

Reply:
Well, I guess it's a somewhat modified truth and probably could be used as a generic template as well.

Reason I called it teams specific is that teams supports limited tags:
https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/bots/bots-text-formats

So if someone decides to update the template in the future maybe they would add tags that were not supported in teams.

Reply:
Hmm, seems like we should have a template collection like we do for alert rules.